File: server/FitMate.Core/JsonModels/AI/StartAIRunResponse.cs
using FitMate.DB.Enums;

namespace FitMate.Core.JsonModels.AI;

public class StartAIRunResponse
{
    public long ConversationId { get; set; }
    public long RunId { get; set; }
    public AIRunStatus Status { get; set; }
    public AIMessageModel UserMessage { get; set; } = null!;
}

File: client/src/types/JsonModels/AI/StartAIRunResponse.ts
import type { JsonModels } from "../../backend";
import type { AIRunStatus } from "../Enums/AIRunStatus";

export interface StartAIRunResponse
{
	conversationId: number;
	runId: number;
	status: AIRunStatus;
	userMessage: JsonModels.AI.AIMessageModel;
}
